gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -1073,6 +1073,23 @@ public GradeSaveResponse saveExcuse(final Long assignmentId, final String studen
 
 		return items;
 	}
+    
+	public List<GbStudentGradeInfo> buildMatrixForGradeComparison(Assignment assignment){
+		// Only return the list if the feature is activated
+		boolean serverPropertyOn = serverConfigService.getConfig("gradebookng.allowStudentsToCompareGradesWithClassmates", false);
```

Comment:
Is there a common property files in GradebookNG? If not no problem

Reply:
I've moved it to the top of the file, but there's no place where the properties are centralized